Makefile

```diff
@@ -1,38 +1,38 @@
 CC ?= gcc
 CFLAGS ?= -O2
-CFLAGS += -std=gnu99 -Wall -pedantic -Werror -fno-strict-aliasing -Wwrite-strings
-LFLAGS += -lmicrohttpd
-SRC = src/main.c src/webserver.c src/utils.c src/files.c src/call.c
+CFLAGS += -std=gnu99 -Wall -Werror -pedantic -fno-strict-aliasing -Wwrite-strings
+LDFLAGS += -lmicrohttpd
+SRC = src/files.c src/utils.c src/webserver.c src/call.c src/main.c
+NUMWWW := `find www/ -type f | wc -l`
```

Comment:
This line does not seem to be used.

Reply:
right. I used it as argument for the g_content array, but then went removing it